docs/diagrams/reset.md

```diff
@@ -0,0 +1,32 @@
+title etcdadm join https://member1:2379
```

Comment:
I think this label is wrong (should be reset)

Reply:
You're right. Copy-paste error. Thanks!